core/controllers/collection_editor_test.py

```diff
@@ -428,7 +428,8 @@ def test_can_search_exploration_with_exploration_id(self):
         exploration_id = exp_fetchers.get_new_exploration_id()
         self.save_new_valid_exploration(exploration_id, self.owner_id)
         rights_manager.publish_exploration(self.owner, exploration_id)
-        exploration_id = base64.b64encode(exploration_id)
+        exploration_id = (
+            base64.b64encode(exploration_id.encode('utf-8')).decode('utf-8'))
```

Comment:
Ditto.

Reply:
Again, we have exploration_id which is `str`, we encode it to `bytes`, then `encode` it to base64 `bytes` and then decode it to base64 `str`.